### PR TITLE
party size filter

### PR DESCRIPTION
**Pull request by Argn0**
closes https://github.com/odota/web/issues/1403

https://github.com/odota/core/pull/1613

### CODE REVIEW

File: src/components/Player/TableFilterForm/index.jsx [hunk 1]
```diff
@@ -204,6 +204,15 @@ class TableFilterForm extends React.Component {
...
               limit={1}
             />
+            <FormField
+              name="party_size"
+              label={strings.filter_party_size}
+              dataSource={data.partySize}
+              formSelectionState={formSelectionState}
+              history={history}
+              strict
+              limit={5}
```
Comment (howardchung) [inline src/components/Player/TableFilterForm/index.jsx]: I think this should be 1

Comment (Argn0) [reply src/components/Player/TableFilterForm/index.jsx]: But what if someone wants all games with party size 2 and party size 1 for example?

Comment (Argn0) [reply src/components/Player/TableFilterForm/index.jsx]: Nevermind, that doesn't work probably.

File: src/components/Player/TableFilterForm/index.jsx [hunk 2]
```diff
@@ -28,7 +28,7 @@ const Styled = styled.div`
 .showForm {
   overflow: hidden;
   transition: max-height 0.2s;
-  max-height: 1000px;
+  max-height: 1200px;
```
Comment (howardchung) [inline src/components/Player/TableFilterForm/index.jsx]: will this just fail if we add more forms?  What if we just remove it?

Comment (Argn0) [reply src/components/Player/TableFilterForm/index.jsx]: OK, removing it works. I tested it.

**Pull request merged**